src/main/java/net/mcreator/ui/SplashScreen.java

```diff
@@ -48,9 +48,9 @@ public SplashScreen() {
 		imagePanel.add(pylo);
 
 		JLabel label = new JLabel(
-				"<html><p>MCreator is a Minecraft mod making toolkit developed by Pylo. Minecraft is a registered</p>"
-						+ "<p style='margin-top:-2'>trademark of Mojang Studios. MCreator is not an official Minecraft product. "
-						+ "It is not approved<br>by or associated with Mojang Studios.");
+				"<html><p>MCreator is a Minecraft mod making toolkit developed by Pylo. Minecraft is a</p>"

```

Comment:
This could be a text block (?)

Reply:
This PR fixes brand name, another PR could focus on using text blocks where needed if in interest